src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3406,10 +3349,7 @@ bool AdaptDispatch::_EraseAll()
         // We don't want to trigger a scroll in pty mode, because we're going to
         // pass through the ED sequence anyway, and this will just result in the
         // buffer being scrolled up by two pages instead of one.
-        if (!inPtyMode)
-        {
-            textBuffer.TriggerScroll({ 0, -delta });
-        }
+        textBuffer.TriggerScroll({ 0, -delta });
```

Comment:
this comment is now out of date - either update to explain why it's OK to do, or remove